lib/util/millisecondsToTimeString.ts

```diff
@@ -1,15 +1,18 @@
-export default function millisecondsToTimeString(ms: number): string {
-  const sec = Math.round((ms / 1000) % 60);
-  const min = Math.floor((ms / 1000 / 60) % 60);
-  const hr = Math.floor(ms / 1000 / 60 / 60);
+import moment from "moment";
+import "moment-duration-format";
 
-  return `${hr}:${min < 10 ? "0" + min : min}:${sec < 10 ? "0" + sec : sec}`;
+export default function millisecondsToTimeString(ms: number): string {
+  // @ts-expect-error
+  return moment.duration(ms).format(
```

Comment:
Great that we can use a proper library for this now instead of my clunky methods :)

Is there a reason why you couldn't fix the type error? I don't like using `ts-expect-error` unless there's absolutely no other way around it.

(Same for the function below)

Reply:
I see that type defs are available for this library, would that solve it?
https://www.npmjs.com/package/@types/moment-duration-format

Reply:
Installing with `npm i --save-dev @types/moment-duration-format` immediately gets rid of the `ts-expect-error` directives!